src/main/java/com/example/demo/TicketBookingApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.example.demo;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TicketBookingApplication implements CommandLineRunner {
+
+	public static void main(String[] args) {
+		SpringApplication.run(TicketBookingApplication.class, args);
+	}
+
+
+	// initializes database, disable on production
```

Comment:
a jak to wyłączyć na produkcji? spring ma taką możliwość

Reply:
wydzieliłbym to do osobnego pliku i użył @Profile